ts/src/test/static/request/woo.json

```diff
@@ -387,12 +387,16 @@
         ],
         "fetchDepositAddress": [
             {
-                "description": "fetch usdt address",
+                "description": "with network",
```

Comment:
@ttodua can we add a few more entries here using common networks like eth/trx/ etc?